VTUL/iawa
# LIBTD-1844: batch_import rake task takes input path as argument

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1844) (:star:)

* Other Relevant Links (https://webapps.es.vt.edu/jira/browse/LIBTD-1817)

# What does this Pull Request do? (:star:)
This PR allows batch_import task take file path instead of file name as input argument relative to the Rails.application.secrets[:iiif_manifest_base_url].

# What's the changes? (:star:)

* Change input argument from file name to file path
* Set up the tmp directories as necessary

# How should this be tested?

* Run as railsapps: bundle exec rake iawa:batch_import["Ms1990_025_Rudoff.index.csv"] 
* Run as railsapps: bundle exec rake iawa:batch_import["Ms1998_022_Young/index.csv"]
* See if the items being imported properly

# Interested parties
Tag (@pmather) interested parties

(:star:) Required fields
